src/views/blocks/attendees/view-link.php

```diff
@@ -11,35 +11,38 @@
  *
  * See more documentation about our Blocks Editor templating system.
  *
- * @since 4.9
- * @since 4.10.8 Renamed template from order-links.php to view-link.php. Updated to not use the now-deprecated
+ * @since   4.9
+ * @since   4.10.8 Renamed template from order-links.php to view-link.php. Updated to not use the now-deprecated
  *               third parameter of `get_description_rsvp_ticket()` and to simplify the template's logic.
- * @since 4.10.9 Uses new functions to get singular and plural texts.
- * @since 4.12.1 Account for empty post type object, such as if post type got disabled. Fix typo in sprintf placeholders.
+ * @since   4.10.9 Uses new functions to get singular and plural texts.
+ * @since   4.12.1 Account for empty post type object, such as if post type got disabled. Fix typo in sprintf placeholders.
+ * @since   TBD Rename Event to Post.
  *
- * @link https://m.tri.be/1amp Help article for RSVP & Ticket template files.
+ * @link    https://m.tri.be/1amp Help article for RSVP & Ticket template files.
  *
- * @version 4.12.1
+ * @version TBD
  *
- * @var Tribe__Tickets__Editor__Template $this
+ * @var Tribe__Tickets__Editor__Template $this       Template object.
+ * @var int                              $post_id    [Global] The current Post ID to which tickets are attached.
+ * @var array                            $attributes [Global] Attendee block's attributes (such as Title above block).
+ * @var array                            $attendees  [Global] List of attendees with attendee data.
  */
 
 if ( ! defined( 'ABSPATH' ) ) {
 	die( '-1' );
 }
 
 $view      = Tribe__Tickets__Tickets_View::instance();
-$event_id  = $this->get( 'post_id' );
-$event     = get_post( $event_id );
-$post_type = get_post_type_object( $event->post_type );
+$post      = get_post( $post_id );
```

Comment:
:no_entry_sign: Overriding WordPress globals is prohibited. Found assignment to $post

<sub>hash: 1d09601aef749996d88b0bfe13314322</sub>